static/index.html

```diff
@@ -38,8 +38,8 @@
                             <th width="150">Assignee</th>
                             <th width="150">Milestone</th>
                             <th width="100">State</th>
+                            <th width="100">Age (in hours)</th>
                             <th>Repo</th>
-                            <th>HTML</th>
```

Comment:
Actually @rushenn i tested it and the click is not affected at all. I do not have the HTML column and its working fine as it should. So we don't need to add it back.